packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -624,6 +625,17 @@ export class RichText extends Component {
 		}
 	}
 
+	willTrimSpaces( html ) {
```

Comment:
This is a nice opportunity to write some unit test against this method, so that we are always sure it's working like expected, WDYT?

Reply:
Good idea! Will it be OK with you to do it in a separate PR, only to make sure we get this one merged today if possible? (to have at least a couple of days of testing until the code freeze)

Reply:
Yup, of course, I will finish testing very soon so that you can merge this one. @hypest